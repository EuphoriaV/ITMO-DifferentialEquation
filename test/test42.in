2.02
(((x - ((7.44 * x) * 3.55)) + ((x + 5.83) - x)) - ((x + 1.89) * (((x * x) - ((x + x) * x)) * (x * 5.91))))
3.15
2.66