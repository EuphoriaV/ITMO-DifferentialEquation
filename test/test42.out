1.4297262